src/controllers/applicant.ts

```diff
@@ -100,7 +101,21 @@ export const listAllApplicants = async (req: Request, res: Response): Promise<vo
         const cookies = Cookie.parse(req.headers.cookie);
 
         const applicants = await Applicant.find({ userId: cookies.googleId });
-        res.status(201).send(applicants);
+        const applicantResponseList = [];
+
+        for (const applicant of applicants) {
+            const screeningGrade = await ScreeningGrade.findById(applicant.screeningGrade);
+            const applicantResponseListObject = {};
+
+            for (const property in applicant['_doc'])
+                applicantResponseListObject[property] = applicant['_doc'][property];
+
+            applicantResponseListObject['screeningGradeActual'] = screeningGrade;
+
+            applicantResponseList.push(applicantResponseListObject);
+        }
+
+        res.status(201).send(applicantResponseList);
```

Comment:
NGL kind of yucky, Its really hard to edit Mongoose response objects so I had to compose a new one (applicantResponseListObject) which included the actual screening grade inside (simplifying the dashboard request object to one API call for sorting) @thomaslapadat